clients/drcachesim/tracer/raw2trace.h

```diff
@@ -961,8 +961,13 @@ class raw2trace_t {
     /**
      * The trace_entry_t buffer returned by get_write_buffer() is assumed to be at least
      * #WRITE_BUFFER_SIZE large.
+     *
+     * Some of the AArch64 SVE scatter/gather instructions have a lot of memref entries.
+     * For example ld4b loads 4 registers with byte sized elements, so that is
+     *     (vl_bits / 8) * 4
+     * entries. With a 512-bit vector length that is (512 / 4 * 8) = 256 memref entries.
      */
-    static const uint WRITE_BUFFER_SIZE = 64;
+    static const uint WRITE_BUFFER_SIZE = 260;
```

Comment:
While testing drcachesim offline I found that we were seeing the `"Too many entries"` error from `raw2trace.cpp:1825` on some of the larger multi-vector register instructions. Increasing the `WRITE_BUFFER_SIZE` seems to fix the issue; is that the right solution?

Reply:
IIRC the buffer only needs to handle records for a single instruction.  But a scatter/gather can have many records so maybe it is not surprising it hit the limit?  But do the SVE ones have more records than x86 ones which did not hit the limit?  @abhinav92003 did you hit this issue on x86?

Yes, if it's slightly larger, increasing makes sense.  I assume it's not 10x larger.

Reply:
> @abhinav92003 did you hit this issue on x86?

On x86 I did not hit this issue on raw2trace. I did hit a similar issue in the tracer though: #6313.

Reply:
It sounds like increasing the buffer size in an appropriate solution.